src/cljc/matthiasn/systems_toolbox/component.cljc

```diff
@@ -92,9 +106,11 @@
         sliding-out-chan (make-chan-w-buf (:sliding-out-chan cfg))
         put-fn (fn [msg]
                  (let [msg-meta (-> (merge (meta msg) {})
-                                    (assoc-in, [:cmp-seq] cmp-id) ; TODO: replace by actual sequence
-                                    (assoc-in, [cmp-id :out-timestamp] (now)))
-                       msg-w-meta (with-meta msg msg-meta)]
+                                    (add-to-msg-seq cmp-id)
+                                    (assoc-in [cmp-id :out-ts] (now)))
+                       corr-id (uuid)
```

Comment:
I'd add some note explaining that tag is a constant thing, while corr-id changes each time message is dispatches. It's clear now as we just discussed it, but might be useful 2 months from now ;)